protocol/x/delaymsg/types/keys.go

```diff
@@ -23,5 +23,7 @@ const (
 
 // Log
 const (
```

Comment:
nit: we should prob create a separate file `types/constants.go` or `types/logs.go`